critiquebrainz/frontend/views/label.py

```diff
@@ -0,0 +1,60 @@
+from flask import Blueprint, render_template, request
+from werkzeug.exceptions import NotFound
+from flask_login import current_user
+from flask_babel import gettext
+import critiquebrainz.db.review as db_review
+import critiquebrainz.frontend.external.musicbrainz_db.label as mb_label
+import critiquebrainz.frontend.external.musicbrainz_db.exceptions as mb_exceptions
+from critiquebrainz.frontend.views import get_avg_rating, LABEL_REVIEWS_LIMIT
+from critiquebrainz.frontend.forms.rate import RatingEditForm
+
+
+label_bp = Blueprint('label', __name__)
+
+
+@label_bp.route('/<uuid:id>')
+def entity(id):
+    id = str(id)
+    try:
+        label = mb_label.get_label_by_id(id)
+    except mb_exceptions.NoDataFoundException:
+        raise NotFound(gettext("Sorry, we couldn't find a label with that MusicBrainz ID."))
+
+    label_reviews_limit = LABEL_REVIEWS_LIMIT
+    if request.args.get('reviews') == "all":
+        label_reviews_limit = None
+
+    if current_user.is_authenticated:
+        my_reviews, my_count = db_review.list_reviews(
+            entity_id=label['id'],
+            entity_type='label',
+            user_id=current_user.id,
+        )
+        my_review = my_reviews[0] if my_count else None
```

Comment:
What happens if I have a review already for a label. It should be passed to the `render_template` to display the option of `Edit Review` rather than `Write Review`. See `place.py` or `release_group.py` for example.

Reply:
Also, other important variable used in the template like `current_user`.

Reply:
All the other important variables too? I think I only passed the ones that we're currently using, to the render template.

Reply:
See the `base.py` (the variable `current_user` and `my_review` do play a role there.

Reply:
I've added current_user and my_review those seem to be the only variables used outside the label template.